vital/types/track_descriptor.cxx

```diff
@@ -153,6 +157,15 @@ double&
 track_descriptor
 ::at( const size_t idx )
 {
+  #ifdef DEBUG_CHECK_DATA_NULL
+  if (!this->data_)
+  {
+    std::stringstream msg;
+    msg << "Attempted to access raw data of null descriptor ptr";
+    throw std::logic_error( msg.str() );
```

Comment:
If the `msg` is just a simple string, you can just put that string in the initializer for `std::logic_error( "Attempted...ptr" );`

And below too.